demo/vl-search.html

```diff
@@ -84,15 +86,39 @@ <h2>Search block met custom label slot</h2>
     </div>
   </div>
 
-  <script>
-    document.querySelectorAll('vl-search').forEach(
-      search => search.addEventListener('change', (event) => {
-        const zoekterm = document.createElement("p");
-        zoekterm.setAttribute('name', 'zoekterm');
-        zoekterm.innerText = `Zoekterm: ${event.srcElement.value}`;
-        search.parentElement.appendChild(zoekterm);
-      }));
-  </script>
+<div>
+  <h2>Search met select</h2>
+  <pre>
+      <code class="line-numbers language-markup">
+        &lt;vl-search id="search-block-slot-label" data-vl-block&gt;
+          &lt;select id="select-default" is="vl-select" slot="input" block data-vl-select&gt;
+            &lt;option value="Belgium"&gt;België&lt;/option&gt;
+            &lt;option value="Germany"&gt;Duitsland&lt;/option&gt;
+            &lt;option value="France"&gt;Frankrijk&lt;/option&gt;
+          &lt;/select&gt;
+        &lt;/vl-search&gt;
+    </code>
+    </pre>
+  <div class="demo">
+    <vl-search id="search-block-slot-input" data-vl-block>
+      <select id="select-default" is="vl-select" slot="input" block data-vl-select>
+        <option value="Belgium">België</option>
+        <option value="Germany">Duitsland</option>
+        <option value="France">Frankrijk</option>
+      </select>
+    </vl-search>
+  </div>
+</div>
+
+<script>
+  document.querySelectorAll('vl-search').forEach(
+    search => search.addEventListener('change', (event) => {
+      const zoekterm = document.createElement("p");
+      zoekterm.setAttribute('name', 'zoekterm');
+      zoekterm.innerText = `Zoekterm: ${event.srcElement.value}`;
+      search.parentElement.appendChild(zoekterm);
+    }));
```

Comment:
waarom staat dat er gewoon niet bij in alle demo's? vind ik een beetje raar dat ge eerst demo-blokken maakt en die dan allemaal nog gaat veranderen

Reply:
Wat bedoel je precies? Die is toch voor elke vl-search? Dus voor elke demo?
Deze code stond er trouwens al.

Reply:
waarom hebt ge een aantal demo's waar ge dan met code nog een stuk aan moet toevoegen. waarom staat die HTML niet gewoon bij in de demo.

Reply:
om die code niet te moeten dupliceren?

Reply:
ik zou hopen dat elke demo "compleet en losstaand" is, buiten misschien wat JS voor functionaliteit die je wil demonsterern. dus ik vind dit een beetje raar.

Reply:
Wil je dat ik die code kopieer naar elke demo?

Reply:
het lijkt mij logischer dat elke demo op zich staat, maar misschien had @coemans of iemand anders die dit gemaakt heeft daar een goeie reden voor?

Reply:
Ah maar wacht, het is niet gewoon statische HTML toevoegen, tis echt ne listener. Nee never mind dan, ik zie het nu.